tests/phpunit/api/v3/EmailTest.php

```diff
@@ -376,4 +376,38 @@ public function testReplaceEmailWithId() {
     $this->assertEquals('1-2@example.com', $get['values'][$emailID]['email']);
   }
 
+  public function testEmailOnHold() {
+    $params = array();
+    $params_change = array();
+    $params = array(
+      'contact_id' => $this->_contactID,
+      'email' => 'api@a-team.com',
+      'on_hold' => '2',
+    );
+    $result = $this->callAPIAndDocument('email', 'create', $params, __FUNCTION__, __FILE__);
```

Comment:
just use callApiSuccess - callAPIAndDocument generates an example file (examples tab in api) so use it when you are demonstrating some special usage or there is not one for the entity